src/command/defaults/ClearCommand.php

```diff
@@ -120,16 +120,16 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 		if($targetItem === null){
 			// Clear all items from the inventories
 			foreach($inventories as $inventory) {
-				$clearedCount += $this->countItems([$inventory], null);
 				$inventory->clearAll();
 			}
+			$clearedCount += $this->countItems($inventories, null);
```

Comment:
This should add to the cleared count before performing the inventory executions, because doing it after the added count will always be 0 since you just cleared the inventories

Reply:
Good catch 😅